R/build_sparql.R

```diff
@@ -8,9 +8,9 @@
 #' spq_add("?city wdt:P1082 ?pop") %>%
 #' spq_language("fr") %>%
 #' spq_head(n=5) %>%
-#' build_sparql() %>%
+#' assemble_sparql_query() %>%
```

Comment:
I found this a better name but I might be wrong, of course.

Reply:
it's fine but maybe a bit long (sorry I comment whereas you're already a bit far in the process of renaming and changing this everywhere...). Would spq_build (and spq_process) sound fine to you?

Reply:
or spq_assemble? I can use your name suggestion next week, easy enough to change, no worries.

Reply:
Ou alors spq_query mais y'a un risque de confondre avec le verbe "query"